src/dag_data.hpp

```diff
@@ -0,0 +1,320 @@
+// Copyright 2019-2022 bito project contributors.
+// bito is free software under the GPLv3; see LICENSE file for details.
+//
+// This class handles DAG General Data stored on nodes or edges. Handles storage and
+// resizing of data vectors.
+
+#pragma once
+
+#include "reindexer.hpp"
+#include "gp_dag.hpp"
+#include "graft_dag.hpp"
+
+// Resizer is a tool for resizing padded data vectors.
```

Comment:
Could we get a little more doc here? There is spare space, padding, alloc, etc. What are these things and how are they different?